### PR TITLE
Add importPath static property to elements

### LINKED ISSUE #97: Use import.meta.url
URLs in CSS were file-relative. After processing with html2js they're main document relative.

There's also the resolveUrl API, which does similar things.

What should we do here?

Failing polymer test to confirm:

unit/custom-style.html
  - style paths in included dom-modules loaded in import

also most of unit/resolveurl.html

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #97

### CODE REVIEW

File: src/document-converter.ts [hunk 1]
```diff
@@ -691,6 +694,65 @@ export class DocumentConverter {
...
+              jsc.identifier('import'), jsc.identifier('meta')),
+          jsc.identifier('url'));
+
+      const node = nodePath.node;
+      if (node.type === 'ClassDeclaration' || node.type === 'ClassExpression') {
+        // A Polymer 2.0 class-based element
+        node.body.body.splice(
+            0,
+            0,
+            jsc.methodDefinition(
```
Comment (FredKSchott) [inline src/document-converter.ts]: pull this into a variable

File: src/document-converter.ts [hunk 2]
```diff
@@ -691,6 +694,65 @@ export class DocumentConverter {
...
+                    null, [], jsc.blockStatement([jsc.returnStatement(
+                                  importMetaUrl)])),
+                true));
+      } else if (node.type === 'CallExpression') {
+        // A Polymer hybrid/legacy factory function element
+        const arg = node.arguments[0];
+        if (arg && arg.type === 'ObjectExpression') {
+          arg.properties.unshift(jsc.property(
+              'init', jsc.identifier('importPath'), importMetaUrl));
+        }
```
Comment (FredKSchott) [inline src/document-converter.ts]: else, emit warning

Comment (justinfagnani) [reply src/document-converter.ts]: done

File: src/document-converter.ts (hunk 1 above)
Comment (justinfagnani) [reply src/document-converter.ts]: done

**Pull request merged**